guides/keras_cv/classification_with_keras_cv.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Title: Classification with KerasCV
+Author: [lukewood](https://lukewood.xyz)
+Date created: 03/28/2023
+Last modified: 03/28/2023
+Description: Use KerasCV to train a state of the art image classifier.
+"""
+
+"""
+KerasCV is a computer vision library that supports users through their entire
```

Comment:
Since this isn't a "getting started" guide for the whole library, I think we can omit these two paragraphs.